osu.Game.Rulesets.Mania/Difficulty/ManiaPerformanceAttributes.cs

```diff
@@ -12,6 +12,12 @@ public class ManiaPerformanceAttributes : PerformanceAttributes
         [JsonProperty("difficulty")]
         public double Difficulty { get; set; }
 
+        [JsonProperty("estimated_ur")]
+        public double? EstimatedUr { get; set; }
```

Comment:
Same as https://github.com/ppy/osu/pull/20963, would prefer this to be called `EstimatedUnstableRate`

Reply:
Sure.